beacon/validator/config.go

```diff
@@ -26,7 +26,7 @@ const (
 
 	// defaultEnableOptimisticPayloadBuilds is the default
 	// for enabling the optimistic payload builder.
-	defaultEnableOptimisticPayloadBuilds = false
+	defaultEnableOptimisticPayloadBuilds = true
```

Comment:
TMP: to be removed, with the rest of changes in commit 517878f007e4635130190db44fd2ad09c34959d4